packages/block-library/src/gallery/gallery-image.js

```diff
@@ -281,9 +281,7 @@ class GalleryImage extends Component {
 					<RichText
 						tagName="figcaption"
 						aria-label={ __( 'Image caption text' ) }
-						placeholder={
-							isSelected ? __( 'Write caption…' ) : null
-						}
+						placeholder={ isSelected ? __( 'Add caption' ) : null }
```

Comment:
I've not seen this pattern before. Why is the placeholder removed on deselect? Normally the whole rich caption is removed.

Reply:
I don't know, but it will likely disappear when the gallery refactor lands

Reply:
Yes, just noticing it, no big deal. :)